conftest.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Configuration of pytest
+"""
+
+import pytest
+from django.conf import settings
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp554ewbzj/conftest.py
+++ b/tmp/tmp554ewbzj/conftest.py
@@ -7,6 +7,7 @@
 
 import pytest
 from django.conf import settings
+
 
 @pytest.fixture(scope='session')
 def django_db_setup():
```